resdx/conditions.py

```diff
@@ -10,21 +10,27 @@ def __init__(self, outdoor=STANDARD_CONDITIONS,
     self.compressor_speed = compressor_speed # compressor speed index (0 = full speed, 1 = next lowest, ...)
     self.rated_air_flow_set = False
 
-  def set_rated_air_flow(self, air_vol_flow_per_rated_cap, net_total_cooling_capacity_rated):
+  def set_rated_air_flow(self, air_vol_flow_rated_per_rated_cap, net_total_cooling_capacity_rated, air_vol_flow_per_rated_cap = None, air_vol_flow = None):
```

Comment:
You shouldn't need to pass "air_vol_flow_per_rated_cap" through here. You have all the information you need before you call this function to only pass in air_vol_flow.